src/tests/functional/plugin/cpu/subgraph_tests/src/any_layout.cpp

```diff
@@ -0,0 +1,91 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "test_utils/cpu_test_utils.hpp"
+
+using namespace InferenceEngine;
+
+class AnyLayoutOnInputsAndOutputs : public ::testing::TestWithParam<ov::Shape> {
```

Comment:
Could you please add `namespace SubgraphTestsDefinitions {` to avoid possible name conflicts and to be consistent with other subgraph tests

Reply:
done